tsz-macro/Cargo.toml

```diff
@@ -14,5 +14,7 @@ version = { workspace = true }
 proc-macro = true
 
 [dependencies]
-quote = "1.0.26"
-syn = { version = "2.0.10", features = ["full"] }
+quote = "1.0.33"
+syn = { version = "2.0.38", features = ["full", "extra-traits"] }
```

Comment:
I don't think that we need full anymore. I could be wrong but I think a lot of those features are just for debugging or testing

Reply:
Updated syn dependency to have only the default features in commit [58a64f9](https://github.com/qsib-cbie/tsz/pull/45/commits/58a64f929b9609d748c4095d2e11e43df4f9b7e8).